src/sap.m/test/sap/m/qunit/SelectList.qunit.js

```diff
@@ -2031,6 +2031,36 @@ sap.ui.define([
 		oSelectList.destroy();
 	});
 
+	QUnit.test("with Delimited, it shouldn't prevent browser from navigating to previous or next page", function(assert) {
```

Comment:
Change the test description:
When keyboardNavigationMode is set to Delimited, it shouldn't prevent browser from navigating to previous or next page.